src/main/java/teammates/common/datatransfer/FeedbackSessionResponseStatus.java

```diff
@@ -67,16 +67,20 @@ public FeedbackSessionResponseStatus() {
     }
 
     /**
-     * Returns list of students who did not respond to the feedback session.
+     * Returns list of students who did not respond to the feedback session
+     * sorted by teamName > studentNamelist.
      */
     public List<String> getNoResponse() {
+        noResponse.sort(compareByTeamNameStudentName);
```

Comment:
Now this does exactly the same as `getStudentsWhoDidNotRespondToAnyQuestion`. Instead of modifying this getter, rename `getStudentsWhoDidNotRespondToAnyQuestion` -> `getStudentsWhoDidNotRespondSorted`. Also, this is a good chance to rename `noResponse` -> `studentsWhoDidNotRespond`.